cli/merge_kubeconfigs.py

```diff
@@ -0,0 +1,38 @@
+import sys
+import yaml
+import json
+
+def _parse_kube_config(filename):
+    try:
+        data = open(filename).read()
+
+        # yaml.load seems to tolerate json too!
+        config = yaml.load(data, Loader=yaml.FullLoader)
+
+        return config
+    except Exception as e:
+        print(e)
```

Comment:
This does not print the full exception. Why not just raise an exception here?

Reply:
Done.